sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/RedundantJumpStatementTest.cs

```diff
@@ -31,7 +32,10 @@ public class RedundantJumpStatementTest
         [TestCategory("Rule")]
         public void RedundantJumpStatement()
         {
-            Verifier.VerifyAnalyzer(@"TestCases\RedundantJumpStatement.cs", new RedundantJumpStatement());
+            Verifier.VerifyAnalyzer(@"TestCases\RedundantJumpStatement.cs",
+                new RedundantJumpStatement(),
+                ParseOptionsHelper.FromCSharp8,
+                additionalReferences: NuGetMetadataReference.NETStandardV2_1_0);
```

Comment:
This `additionalReferences` is not necessary here since we upgarded the lang version of the unit test project to 8 (#2886).
There is even a ticket to remove this logic now that we can (#2802).

Reply:
This one cannot be removed, it uses default interface implementation.